components.cake

```diff
@@ -114,7 +114,8 @@ void SetArtifactsPodSpecs ()
 		PodSpec.Create ("gRPC-C++",       "0.0.9",  frameworkSource: FrameworkSource.Pods, frameworkName: "grpcpp"),
 	};
 	FIREBASE_CLOUD_MESSAGING_ARTIFACT.PodSpecs = new [] {
-		PodSpec.Create ("Firebase", "6.5.0", frameworkSource: FrameworkSource.Pods, frameworkName: "FirebaseMessaging", targetName: "FirebaseMessaging", subSpecs: new [] { "Messaging" })
+		PodSpec.Create ("Firebase", "6.5.0", frameworkSource: FrameworkSource.Pods, frameworkName: "FirebaseMessaging", targetName: "FirebaseMessaging", subSpecs: new [] { "Messaging" }),
+		PodSpec.Create ("Protobuf", "3.8.0", frameworkSource: FrameworkSource.Pods, canBeBuild: false),
```

Comment:
Is it worth creating a separate binding for this? Do any other components pull this in - potentially causing duplicate references or something?

Reply:
what should I do as a developer? i can not see the 4.1.1.1 version on nuget, how to use this fixed version?

Reply:
How can I get this fix installed? I can't see 4.1.1.3 version of the Firebase.iOS.CloudMessaging available even when I check show pre-release-packages